requirements-server.txt

```diff
@@ -4,12 +4,12 @@
 # For generating readthedocs
 -r requirements.docs.txt
 
+# Only for development
+-r requirements.dev.txt
```

Comment:
Do we need to copy the file over to the container?

https://github.com/codalab/codalab-worksheets/blob/2f36fecfc351dea17e01c6b3e040483a7c0c94cd/docker/dockerfiles/Dockerfile.server#L37

Reply:
What would happen if we just set the version here and not require `requirements.dev.txt`?

Reply:
We could do that, but then we would have to store the version in two spots -- one, this file, and two, in CI. It seems better to have a single version in requirements.dev.txt so it's easy to bump and we will never have the versions out of sync.